.github/workflows/create_containers.yml

```
@@ -29,36 +40,81 @@ jobs:
         uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
         with:
           persist-credentials: false
+      - name: Assemble container tag
+        id: tag
+        env:
+          DISTRO: ${{ matrix.distros.DISTRO }}
+          ARCH: ${{ matrix.distros.ARCH }}
+          VARIANT: ${{ matrix.distros.VARIANT }}
+          CONTAINER_TAG: ${{ env.CONTAINER_TAG }}
+          USER_REPO: ${{ github.event.pull_request.head.repo.full_name }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:44: Double quote to prevent globbing and word splitting

(shellcheck)